src/odemis/odemisd/start.py

```diff
@@ -655,7 +655,7 @@ def main(args):
     except ValueError:
         loglevel = 1
         odemis_config["LOGLEVEL"] = "%d" % loglevel
-    logging.getLogger().setLevel(loglevel)
+    logging.getLogger().setLevel(40)
```

Comment:
It should be changed but I think that in the lines above a small erorr is present.
The levels in log level are actually a multiple of 10. (So 10=DEBUG, 20=INFO, 40=ERROR)
The current implementation sets the logging level to 1, 2, 3 or 4 meaning it will always be in DEBUG mode. I think we want to change that isn't it?

Reply:
https://www.geeksforgeeks.org/logging-in-python/
![image](https://user-images.githubusercontent.com/58170830/172554782-707d5104-ee04-4eb2-91a9-2c78e0f5edc6.png)